app/src/main/java/com/example/workoutapp/RetrofitClientInstance.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.workoutapp
```

Comment:
added missing RetrofitClientInstance file. Implementing this and DAO and DTO packaged should help make unit tests run.